internal/jsonschema/jsonschema.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021-2023 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package jsonschema
+
+import (
+	"fmt"
+	"io"
+	"io/fs"
+	"log"
+
+	"github.com/santhosh-tekuri/jsonschema/v5"
+	"gopkg.in/yaml.v3"
+
+	"github.com/cerbos/cerbos/schema"
+)
+
+var (
+	policySchema *jsonschema.Schema
+	testSchema   *jsonschema.Schema
+)
+
+func init() {
+	var err error
+	if policySchema, err = jsonschema.CompileString("Policy.schema.json", string(schema.PolicyJSONSchema)); err != nil {
+		log.Fatalf("failed to compile policy schema: %v", err)
+	}
+
+	if testSchema, err = jsonschema.CompileString("TestSuite.schema.json", string(schema.TestSuiteJSONSchema)); err != nil {
+		log.Fatalf("failed to compile test schema: %v", err)
+	}
+}
+
+// ValidatePolicy validates the policy in the fsys with the JSON schema.
+func ValidatePolicy(fsys fs.FS, path string) error {
+	return validate(policySchema, fsys, path)
+}
+
+// ValidateTest validates the test in the fsys with the JSON schema.
+func ValidateTest(fsys fs.FS, path string) error {
```

Comment:
This isn't used anywhere. I assume you still haven't gotten around to it?

Reply:
It's actually used in `internal/verify/verify.go:101` ([link](https://github.com/cerbos/cerbos/pull/1526/files#diff-7285c90794dc46994c4a2590334d2c9da31710e20af9a125dd479e997ae0e5bdR101))

Reply:
My bad. Sorry.